core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/details/NFCDocumentTag.kt

```diff
@@ -137,6 +137,36 @@ class NFCDocumentTag(val readDG2: Boolean = true) {
                     additionalPersonDetails.title = dg11.title
 
                     passport.additionalPersonDetails = additionalPersonDetails
+
+                    // Hash Checking
+                    val hashCheckNotSucceeded = "hash-check not SUCCEEDED"
+                    if (verifySecurity.ht != VerificationStatus.Verdict.SUCCEEDED) {
+                        Log.e(TAG, hashCheckNotSucceeded)
+                    }
+
+                    // when enabled, send logs to Sentry
+                    if (captureLog) {
+                        // Send to Sentry hashCheckNotSucceeded
+                        Sentry.captureMessage(hashCheckNotSucceeded)
```

Comment:
We only log  `hashCheckNotSucceeded` message if `verifySecuriy.ht != VerificationStatus.Verdict.SUCCEEDED`